features/polyExplorer/src/screens/dataExploration/dataExploration.jsx

```diff
@@ -58,9 +58,7 @@ const DataExplorationScreen = () => {
     // the -1 is because in the moment of click the activeindex is increased by swiper
     // these problems will solve themselves when switching to scrollytelling
     const saveActiveIndex = () => {
-        history.entries[
-            history.entries.length - 2
-        ].state.explorationState.index = activeIndex;
+        navigate[navigate - 2].state.explorationState.index = activeIndex;
```

Comment:
Double check that this works - `navigate` is supposed to be a function, not an array.